buildSrc/src/main/java/gradlebuild/JniPlugin.java

```diff
@@ -144,6 +144,7 @@ private void configureNativeJars(Project project, VariantsExtension variants, bo
         // We register the publications here, so they are available when the project is used as a composite build.
 // When we don't use the software model plugins anymore, then this can move out of the afterEvaluate block.
         project.afterEvaluate(ignored -> {
+            String artifactId = project.getTasks().named("jar", Jar.class).get().getArchiveBaseName().get();
```

Comment:
Why not use the project's name?

Reply:
Because we are not using the project name in `testApp`.